### DISCUSSION TITLE
Tooltip for long table names in Table Editor

### DISCUSSION THREAD

**Discussion by saltcod** [supabase/supabase#17270] [Contribute to Supabase]
If we have long table names, there's no way to read them after they've been truncated. We should use tooltip here to show the full names on hover.

![screenshot-2023-09-08-at-09 04 53](https://github.com/supabase/supabase/assets/105593/4f8d0e29-efa3-4d7e-87dd-e15598f504e9)

**Comment by lorenzejay** [comment]
I'll give this a shot!

**Comment by lorenzejay** [comment]
This seems to be resolved already. Or is this not it?


<img width="425" alt="Screenshot 2023-10-17 at 11 20 05 AM" src="https://github.com/supabase/supabase/assets/63378463/a3e96746-08f6-479d-ad25-8c18e3eddde7">

<img width="1037" alt="Screenshot 2023-10-17 at 11 20 40 AM" src="https://github.com/supabase/supabase/assets/63378463/97999579-9be1-48e4-8eca-d1cd84fa0cd2">

**Comment by lorenzejay** [comment]
Or does tooltip to the right make more sense?

<img width="603" alt="Screenshot 2023-10-17 at 11 29 40 AM" src="https://github.com/supabase/supabase/assets/63378463/84c8f58e-0c8f-4f1a-a0f3-a08ff0da5a65">

**Comment by lorenzejay** [comment]
@saltcod wdyt ?

**Comment by saltcod** [comment]
Yes it is. Thanks @lorenzejay
